composer.json

```diff
@@ -44,6 +44,9 @@
         "phpunit/phpunit": "~6.0",
         "symfony/dom-crawler": "~3.4"
     },
+    "_readme": [
+        "We explicitly require drupal/inline_entity_form version to be higher than rc7 to avoid known issues with the module."
+    ],
```

Comment:
The comment is too generic, we require rc8 because we need the bug fix in it, more than "known issues". Let's improve the comment.